templates/split.html

```
@@ -43,7 +50,7 @@ <h3>{{ _("Track 1") }}</h3>
                     <span class="track-label">{{ _("MXM Link") }}</span>
                     <span class="track-value">
                       <a href="{{ split_result.track1.track_share_url.split(" ?")[0] }}"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)